### PR TITLE
MCKIN-21918 Inline discussion japanese buttons

### PR DESCRIPTION
**Pull request by msaqib52**
This is replacement PR for https://github.com/edx-solutions/edx-platform/pull/1888, because ci tests were not running on that PR. Code changes are made by @murad-hubib

### CODE REVIEW

**Review by ihtram — APPROVED** [review]
LGTM. Please wait for ci tests to complete before merging

**Review by shafqatfarhan — APPROVED** [review]
👍

**Pull request merged**
